rust_src/README.md

```diff
@@ -48,10 +48,10 @@ use libc::{c_double, /* ... */ };
 // ...
 ```
 
-Second, `Lisp_Object` is an alias to `EmacsInt`:
+Second, `Lisp_Object` is a struct containing an EmacsInt:
```

Comment:
It currently is an alias.

Reply:
Hmm. Not in Wilfred/master, unless I'm missing something:
```
rust_src/remacs-sys/lib.rs:pub struct Lisp_Object(EmacsInt);
```

Reply:
As of #562 it's not an alias.

Reply:
Yeah, my bad. That was even my PR too. Sigh. I have swapped out remacs for the work stack.